src/sentry/api/event_search.py

```diff
@@ -169,9 +169,14 @@
 iso_8601_date_format = date_format time_format? ("Z" / tz_format)? &end_value
 rel_date_format      = ~r"[+-][0-9]+[wdhm]" &end_value
 duration_format      = numeric ("ms"/"s"/"min"/"m"/"hr"/"h"/"day"/"d"/"wk"/"w") &end_value
```

Comment:
Can we match what we did in the pegjs grammar and pull out the `duration_unit` too? I was trying to keep these as close to the same as possible

Reply:
Yup, I'll make all these units separate expression in the next PR when I update the grammar to not be case sensitive